modules/prediction/common/feature_output.cc

```diff
@@ -129,6 +138,19 @@ void FeatureOutput::WritePredictionResult() {
   }
 }
 
+void FeatureOutput::WriteFrameEnv() {
+  if (list_frame_env_.frame_env_size() <= 0) {
+    ADEBUG << "Skip writing empty prediction_result.";
+  } else {
+    const std::string file_name =
+        FLAGS_prediction_data_dir + "/frame_env." +
+        std::to_string(idx_frame_env_) + ".bin";
```

Comment:
StrCat(FLAGS_prediction_data_dir, "/frame_env.", idx_frame_env_, ".bin")

Current style will generate many temporary string objects.

Reply:
Will fix it in the next commit, Thanks.